app.rambox.ramboxce.yaml

```diff
@@ -34,7 +34,6 @@ modules:
     sources:
       - type: file
         dest-filename: rambox.deb
-        only-arches: [x86_64]
```

Comment:
I think removing `only-arches` was tied into building this from source; as long as this is extracted from the debian package, though, doesn't it need to remain?